webextension/scripts/popup.js

```diff
@@ -1,15 +1,15 @@
 // popup.js
 
 // from 'utils.js'
-/*   global isValidUrl, isNotExcludedUrl, openByWindowSetting, hostURL, feedbackPageURL, newshosts, dateToTimestamp */
+/*   global isValidUrl, isNotExcludedUrl, openByWindowSetting, hostURL, feedbackPageURL, newshosts, dateToTimestamp, searchValue */
 
 function homepage() {
   openByWindowSetting('https://web.archive.org/')
 }
 
 function save_now() {
   chrome.tabs.query({ active: true, currentWindow: true }, (tabs) => {
-    let url = tabs[0].url
+    let url = searchValue.length >= 3 ? searchValue : tabs[0].url
```

Comment:
'3' could be a const assigned at the start of the file.

Reply:
I found a way to make it work without using it.
Thank you!